bin/helpers/sync/syncSpecsLogs.js

```diff
@@ -105,6 +104,7 @@ let whileProcess = (whilstCallback) => {
         whileLoop = false;
         endTime = Date.now();
         showSpecsStatus(body);
+        logger.info("\n--------------------------------------------------------------------------------")
```

Comment:
Why are we removing the above table config and adding static lines in logs here?

Reply:
According to the spec , we don't need table config's border.